src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -72,6 +72,20 @@ public CollectionItemsEqualConstraint Using(IComparer comparer)
             return this;
         }
 
+#if (NET_4_0 || NET_4_5) && !PORTABLE
```

Comment:
I wasn't sure if the portable framework had the Func delegate. In truth, I'm not sure how to test the code locally to find out. Do you have instructions for testing locally against the various framework versions? I haven't found this information in the documentation yet.